common/src/main/java/whocraft/tardis_refined/common/tardis/themes/DesktopTheme.java

```diff
@@ -4,25 +4,36 @@
 import net.minecraft.resources.ResourceLocation;
 import whocraft.tardis_refined.constants.ModMessages;
 
+import java.util.Optional;
+
 
 public class DesktopTheme {
 
     private final ResourceLocation uiTexture;
     public boolean availableByDefault = false;
-    public String id = "";
+    public ResourceLocation identifier;
+    public String name = "";
     public ResourceLocation location;
```

Comment:
Can we have a clearer name for the "location" field? Location is very vague and may be easily confused, especially when it is then called "structure" in the JSON.

Reply:
Renamed to be more clear